resources/nats/values.yaml

```diff
@@ -330,4 +330,20 @@ commonAnnotations: {}
 exporter:
   enabled: true
   pullPolicy: IfNotPresent
-  resources: {}
+  resources:
+    limits:
+      cpu: 50m
+      memory: 32Mi
+    requests:
+      cpu: 10m
+      memory: 20Mi
+
+# NATS config reloader sidecar.
+reloader:
+  resources:
+    limits:
+      cpu: 50m
+      memory: 32Mi
+    requests:
+      cpu: 10m
+      memory: 20Mi
```

Comment:
Set these values based on consumption in k8s: https://github.com/kyma-project/nats-manager/issues/104#issuecomment-1882614305